astropy/units/equivalencies.py

```diff
@@ -263,6 +267,8 @@ def iconverter_phot_f_nu_to_f_la(x):
         (phot_L_la, phot_L_nu, converter_phot_L_la_phot_L_nu, iconverter_phot_L_la_phot_L_nu),
         (phot_L_nu, L_nu, converter_phot_L_nu_to_L_nu, iconverter_phot_L_nu_to_L_nu),
         (phot_L_nu, L_la, converter_phot_L_nu_to_L_la, iconverter_phot_L_nu_to_L_la),
+        # surface brightness
+        (S_la, S_nu, converter, iconverter),
```

Comment:
cc @mhvk for the new unit equivalency.

Reply:
The addition itself is fine, but I think it would make sense to do also `phot_S_*`, `nu_S_nu`, etc., as for flux and luminosity.

Reply:
I agree with @mhvk as I failed to convert from `BlackBody` flux output to PHOTLAM/sr. I get `UnitConversionError: 'erg / (cm2 Hz s sr)' and 'ph / (Angstrom cm2 s sr)' are not convertible`.

I have to work around it by manually getting rid of `sr`.

Reply:
I have added in all the equivalent surface brightness conversions to the existing flux and luminosity.  I have not added tests for all of them as they use the same converters so that flux/luminosity versions of the tests should be good (I think).  If tests are needed, I can add them through extensive copy and paste from existing tests (or maybe there is a better way).